todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,51 @@
+import os
+import dotenv
+from bson import ObjectId
+import pymongo
+
+from todo_app.data.Item import Item
+
+dotenv.load_dotenv()
+
+def get_collection():
+    """
+    Sets up a conection to the MongoDB database and returns the to do collection.
+    """
+    
+    client = pymongo.MongoClient(os.getenv("MONGODB_CONNECTION_STRING"))
+    db = client[os.getenv("MONGODB_DATABASE_NAME")]
+    return db[os.getenv("MONGODB_COLLECTION_NAME")]
```

Comment:
You may notice that your app is a bit slower since we introduced Mongo, and the main reason is that every single action creates a new connection to the DB

Using the singleton pattern here might be a simple (and certainly not everyone's preferred!) solution - if you're curious try using this approach and see how the app speeds up locally.

```python
shared_collection = None
def get_collection():
    """
    Sets up a conection to the MongoDB database and returns the to do collection.
    """
    global shared_collection
    if shared_collection is not None:
        return shared_collection
    
    client = pymongo.MongoClient(os.getenv("MONGODB_CONNECTION_STRING"))
    db = client[os.getenv("MONGODB_DATABASE_NAME")]
    shared_collection = db[os.getenv("MONGODB_COLLECTION_NAME")]
    return shared_collection
```

In most real codebases, we'd probably solve this in a more OOP fashion, by wrapping these functions in an `ItemService` class or similar, and having the collection be a field on the class so that we can use a shared connection between requests